Explain RULELABEL AUTO option in GetLegendGraphics (#5958)
qgis/QGIS-Documentation: docs/server_manual/services.rst

* **LAYERFONTITALIC / ITEMFONTITALIC** ``TRUE`` to use italic font
* **LAYERFONTCOLOR / ITEMFONTCOLOR** Hex color code (e.g. ``#FF0000``
  for red)
* **LAYERTITLE** ``FALSE`` to get only the legend graphics without layer title
* **RULELABEL**:

>     * ``FALSE`` get only the legend graphics without item labels
>     * ``AUTO`` hide item label for single legend and display item labels for any other legend type. This is useful to avoid having duplicated layer title and item label in case of single legend.

Reviewer: ```suggestion
    * ``FALSE`` legend graphics without item labels
    * ``AUTO`` hide item label if the legend consists of only one item
```

Author: Actually AUTO value is hiding labels only for single legend type and not if there is one (or more) item in legend and legend is other type like categorized, rule label or graduated. Because only in case of single legend label item is equal to layer title and so not needed.

Reviewer: Thanks! I have updated my suggestion to make that more clear.

Author: OK, Do you think that with AUTO value it is clear to the user that in other cases than single legend, labels are visible?

Reviewer: Yes, I think that is obvious.

TILEMATRIXSET
^^^^^^^^^^^^^

> This parameter allows to define the CRS to use to compute the
> underlying pyramid and has to be formed like ``EPSG:XXXX``.

Reviewer: ```suggestion
This parameter defines the CRS to use when computing the underlying
pyramid. Format: ``EPSG:XXXX``.
```
